dlp/api/Snippets.Tests/InspectStorageWithSamplingTests.cs

```diff
@@ -0,0 +1,32 @@
+﻿// Copyright 2023 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using Google.Cloud.Dlp.V2;
+using Xunit;
+
+namespace GoogleCloudSamples
+{
+    public class InspectStorageWithSamplingTests : IClassFixture<DlpTestFixture>
+    {
+        private DlpTestFixture _fixture;
+        public InspectStorageWithSamplingTests(DlpTestFixture fixture) => _fixture = fixture;
+
+        [Fact]
+        public void TestStorageWithSampling()
+        {
+            var result = InspectStorageWithSampling.Inspect(_fixture.ProjectId, _fixture.GcsFileUri, _fixture.TopicId, _fixture.SubscriptionId);
+            Assert.Equal(DlpJob.Types.JobState.Done, result.State);
+        }
```

Comment:
This doesn't even build, you also need to await here.
```suggestion
        public async Task TestStorageWithSampling()
        {
            var result = await InspectStorageWithSampling.InspectAsync(_fixture.ProjectId, _fixture.GcsFileUri, _fixture.TopicId, _fixture.SubscriptionId);
            Assert.Equal(DlpJob.Types.JobState.Done, result.State);
        }
```

Reply:
Yes, The code is working now. Thanks for the resolution. I will make these changes in other PRs as well